docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -436,6 +436,255 @@
     "tab.mean(axis=1)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fe565b65-fbf2-47ba-a26e-791d09fd4f55",
+   "metadata": {},
+   "source": [
+    "### Table.kurt()\n",
+    "\n",
+    "```\n",
+    "Table.kurt(axis=0, skipna=True, numeric_only=False)\n",
+    "```\n",
+    "\n",
+    "Return unbiased kurtosis over requested axis. Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of normal == 0.0). Normalized by N-1.\n",
+    "\n",
+    "\r\n",
+    "**Parameters:**\r\n",
+    "\r\n",
+    "| Name         | Type | Description                                                                      | Default |\r\n",
+    "| :----------: | :--: | :------------------------------------------------------------------------------- | :-----: |\r\n",
+    "| axis         | int  | Axis for the function to be applied on. 0 is columns, 1 is rows.                    | 0       |\r\n",
+    "| skipna       | bool | not yet implemented                                           | True    |\r\n",
+    "| numeric_only | bool | Only use columns of the table that are of a numeric data type.                   | False   |\r\n",
+    "\r\n",
+    "**Returns:**\r\n",
+    "\r\n",
+    "| Type       | Description                                                                              |\r\n",
+    "| :--------: | :--------------------------------------------------------------------------------------- |\r\n",
+    "| Dictionary | Map of columns and their yielded kurtosis values |osis values |"
```

Comment:
"` |osis values `" I think this is a typo

Reply:
Done